[MoM] Nether Attunement Spoilers document (#80857)
CleverRaven/Cataclysm-DDA: data/mods/MindOverMatter/NetherAttunementSpoilers.md

# Nether Attunement
Nether attunement is gained primarily from activating or concentrating on powers. Nether attunement is tracked via a vitamin and each rank of nether attunement corresponds to a certain amount of the attunement vitamin. Overall gain is not linear, when you have no attunement ranks, vitamin gain is slower but when you have any ranks of attunement, you gain more vitamin, faster.

## Chance to Gain Attunement

> Every time you activate a power, you have a chance to gain attunement vitamin based on the difficulty of the power, how many powers you activated recently and how many powers you are concentrating on. When you have no ranks of attunement, the chance to gain attunement vitamin based on recent power usage is lower.

Reviewer: ```suggestion
Every time you activate a power, you have a chance to gain attunement vitamin based on the difficulty of the power, how many powers you activated recently and how many powers you are concentrating on. The base change is the difficulty of the power squared, modified by the other factors. When you have no ranks of attunement, the chance to gain attunement vitamin based on recent power usage is lower.
```

## Amount of Attunement Gained
If this check passes, you then gain some amount of attunement vitamin that is controlled by a variety of factors. When you have no ranks of attunement, you gain attunement vitamin based on the number of powers you are concentrating on plus a small random amount. When you have any ranks of attunement, you will gain attunement vitamin based on the difficulty of the power, your attunement rank and the number of powers you are concentrating on. This latter part is why you gain later attunement ranks faster, the more attunement ranks you have the more attunement vitamin you gain in a self amplifying cycle.

> When you have torrential channeling or noetic resilience active, you will gain less attunement vitamin from power usage and the attunement vitamin you do gain is not affected by your current attunement rank, protecting you from the self amplifying effect of attunement.

Reviewer: ```suggestion
When you have extended channeling or noetic resilience active, you will gain less attunement vitamin from power usage and the attunement vitamin you do gain is not affected by your current attunement rank, protecting you from the self amplifying effect of attunement.
```

|  11  | 2.10 | 1.05 | 2.10 |
|  12  | 3.00 | 1.05 | 3.00 |
|  12* | 4.00 | 1.05 | 4.00 |

There is actually a vitamin level past attunement rank 12 where your attunement power scaling goes up to 400%. You have no visual indicator of reaching this point.

## Attunement Consequences
Whenever you activate a power and you have any ranks of attunement, you will trigger a possible consequence. The specific consequence is pulled from a weighted list, shown below, and if your attunement vitamin is not high enough, a new selection is made from the same list until one is selected that you do have the minimum attunement vitamin for. Some of the consequences are power or power path specific and if you select one that does not apply, a new one will be selected as before.
Once a consequence is selected, it has a chance to actually be applied. There is a base percentage chance plus an increased chance based on two factors: 
- Current attunement vitamin
  - The higher your attunement vitamin, the greater the chance. This chance is scaled at different rates depending on how high your attunement vitamin is and varies from consequence to consequence.
- Difficulty of power used

>   - The formula for this `(difficulty / 10)^2` and is the same for every consequence. Stronger powers come with more chance at consequences

Reviewer: ```suggestion
  - The formula for this `(difficulty ^ 2) / 10` and is the same for every consequence. Stronger powers come with more chance at consequences
```
Realized when I read this that "plus 1/10th the Difficulty squared" is ambiguous

### Nether Conduit
Minimum Attunement: 3  
Chance: 3% to 38.25%  
Description: You gain an effect that unlocks nastier consequences and slowly increases your attunement vitamin for a random period of time based on your attunement vitamin  
<sub>[Back to List](#attunement-consequences)</sub>

### Feedback
Minimum Attunement: 5  
Chance: 3% to 39.5%  
Description: You gain an effect that causes some amount of pain whenever you use your powers that lasts for a random period of time based on your attunement vitamin  
<sub>[Back to List](#attunement-consequences)</sub>

### Observed
Minimum Attunement: 4  
Chance: 2% to 29%  
Description: You gain an effect that unlocks nastier consequences, comes with nightmares that increase your attunement vitamin and lasts a long time based on your attunement vitamin  
<sub>[Back to List](#attunement-consequences)</sub>

### Incorporeality
Minimum Attunement: 4  
Chance: 4% to 37%  
Description: When using the ephemeral walk power, you become incorporeal for a brief period based on your attunement vitamin  
<sub>[Back to List](#attunement-consequences)</sub>

### Teleport Lock
Minimum Attunement: 4  
Chance: 2% to 33.5%  

> Description: You are unable to use your teleporter powers for a brief period based on your attunement vitamin  

Reviewer: ```suggestion
Description: You are unable to teleport yourself for a brief period based on your attunement vitamin  
```
Technically you can still use powers like Stutterstep or Spacial Vortex

<sub>[Back to List](#attunement-consequences)</sub>

### Weakness
Minimum Attunement: 5  
Chance: 2% to 31%  
Description: You have lowered strength and dexterity along with increased stamina usage in melee for a short period based on your attunement vitamin  
<sub>[Back to List](#attunement-consequences)</sub>

### KCal Consumption
Minimum Attunement: 5  
Chance: 4% to 38%  
Description: You lose some of your stored KCal  
<sub>[Back to List](#attunement-consequences)</sub>

### Attenuation
Minimum Attunement: 6  
Chance: 4% to 25.5%  
Description: Your powers all take four times as long to activate for a short period of time based on your attunement vitamin  
<sub>[Back to List](#attunement-consequences)</sub>

### Short of Breath
Minimum Attunement: 5  
Chance: 3% to 25.5%  
Description: You have trouble breathing, lowering your breathing and lifting scores and cough occasionally for a short period based on your attunement vitamin  
<sub>[Back to List](#attunement-consequences)</sub>

### Force Wave
Minimum Attunement: 6  
Chance: 3% to 30%  

> Description: A wave of force knocks you and everything around you down  

Reviewer: ```suggestion
Description: When using a telekinetic power, a wave of force knocks you and everything around you down  
```

<sub>[Back to List](#attunement-consequences)</sub>

### EMP Blast
Minimum Attunement: 8  
Chance: 2% to 18.75%  
Description: Only applicable if you have the Photokinetic power path. You emit an EMP blast centered on yourself  
<sub>[Back to List](#attunement-consequences)</sub>

### Lightning Blast
Minimum Attunement: 9  
Chance: 3% to 19.25%  
Description: You emit a blast of lighting, electrocuting yourself and your surroundings  
<sub>[Back to List](#attunement-consequences)</sub>

### Nullified
Minimum Attunement: 7  
Chance: 2% to 18.25%  
Description: All of your active powers are cancelled and you are unable to activate any more powers for a period of time based on your attunement vitamin  
<sub>[Back to List](#attunement-consequences)</sub>

### Hounds of Tindalos
Minimum Attunement: 10  
Chance: 3% to 8%  
Description: This consequence can only activate if you have the Observed or Nether Conduit effect. You summon the Hounds of Tindalos 
<sub>[Back to List](#attunement-consequences)</sub>
 
### Mutation
Minimum Attunement: 12  
Chance: 3% to 6%  

> Description: You gain a random negative mutation  

Reviewer: ```suggestion
Description: You gain a random negative mental or psychological mutation  
```
